components/shooter.py

```diff
@@ -12,10 +12,11 @@
 
 
 class ShooterComponent:
+    MAX_FLYWHEEL_SPEED = 6380 / 60
```

Comment:
This looks suspiciously like a motor free speed. Don't we have that defined in `utilities.ctre` already?